.github/workflows/testadapters.yaml

```diff
@@ -82,23 +82,17 @@ jobs:
               echo '{ "meshery-provider": "None", "token": "" }'  > ~/.meshery/auth.json
             fi
             cat ~/.meshery/auth.json
-        
+
+      # Add adapter to meshconfig which will be picked up by mesheryctl to start the adapter
+      - name: Add adapter to meshconfig
+        run: |
+            yq -e -i '.contexts.local.components += ["meshery-${{ inputs.adapter_name }}"]' ~/.meshery/config.yaml
```

Comment:
Perhaps, a more straight-forward approach is to create the meshconfig with specific adapters included (upon creation, like so `mesheryctl system context create -a <adapter-name>`).

Reply:
Could not get this to work. 
```
Error: accepts 1 arg(s), received 0
```

Alternative is to create another context and add the adapter to it and switch contexts

Reply:
@gr455  - - 


```
$ mesheryctl system context create test -a meshery-istio
Added `test` context

$ mesheryctl system context view test

Current Context: test

endpoint: http://localhost:9081
token: Default
token-location: auth.json
platform: kubernetes
components:
- meshery-istio
channel: stable
version: latest
```